src/components/Menu/Menu/Menu.jsx

```diff
@@ -1,18 +1,18 @@
-import { SIZES } from "constants/sizes";
+import { SIZES } from "../../../constants/sizes";
 import React, { useMemo, forwardRef, useState, useRef, useEffect, useLayoutEffect, useCallback } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
-import useMergeRefs from "hooks/useMergeRefs";
-import useClickOutside from "hooks/useClickOutside";
-import { backwardCompatibilityForProperties } from "helpers/backwardCompatibilityForProperties";
+import useMergeRefs from "../../../hooks/useMergeRefs";
+import useClickOutside from "../../../hooks/useClickOutside";
+import { backwardCompatibilityForProperties } from "../../../helpers/backwardCompatibilityForProperties";
 import useSubMenuIndex from "./hooks/useSubMenuIndex";
 import useOnCloseMenu from "./hooks/useOnCloseMenu";
 import useCloseMenuOnKeyEvent from "./hooks/useCloseMenuOnKeyEvent";
 import useMenuKeyboardNavigation from "./hooks/useMenuKeyboardNavigation";
 import useMouseLeave from "./hooks/useMouseLeave";
 import "./Menu.scss";
 import { useAdjacentSelectableMenuIndex } from "./hooks/useAdjacentSelectableMenuIndex";
-import { useFocusWithin } from "hooks/useFocusWithin";
+import { useFocusWithin } from "../../../hooks/useFocusWithin";
```

Comment:
@hadasfa lets make sure that we align it to be default export

Reply:
same